.gitignore

```diff
@@ -1,5 +1,36 @@
+#Flask
 flask/env/
 *.pyc
 __pycache__
 photo2code-secret.json
+
+#React
+
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+
+# dependencies
+react-app/node_modules
+react-app/.pnp
+react-app/.pnp.js
+
+# testing
+react-app/coverage
+
+# production
+react-app/build
+
+# misc
+react-app/.DS_Store
+react-app/.env.local
```

Comment:
can we just add `react-app/.env.*` instead of all the various .env denominations?

Reply:
good idea!